blog/_posts/2020-06-18-the-right-display-for-now.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: "the-right-display-for-now"
+date: 2020-06-18
+author: nick
+layout: blog
+title: "The Right Display for Now"
+image: "/assets/images/the-right-display-for-now/forest-park.jpg"
+tags:
+  - tilt
+  - ui
+---
+
+At Tilt, [Han and I](https://tilt.dev/about) talk a lot about the different
+modes developers are in and problems they're solving when using tools. Sometimes
+it's easier to talk about these modes when we relate them to signs in the real
+world.
+
+For example:
+
+- Discovery - what is it possible to do here? (see: the Forest Park sign above)
+- Awareness - what issues should I be aware of? (see: the estimated arrival time signs at train stations)
+- Recommendations - what should I do next? (see: exit signs, recommended routes)
+
+Tilt has always initialized your terminal with a terminal-based GUI. 
+
+We've found that the terminal-based GUI is good for awareness of all your
+different services. But if you're not actively swimming in the deep end of the
+Kubernetes ecosystem, the terminal GUI does a poor job communicating what you
+can do and giving you a recommended path.
+
+In Tilt v0.15.0, we've changed that.
+
+## What Tilt Looks Like Now
+
+When you start Tilt in a terminal, here's what you'll see:
+
+```
+Tilt started on http://localhost:10350/
+v0.15.0, built 2020-06-18
+
+(space) to open the browser
+(s) to stream logs (--hud=false)
+(h) to open terminal HUD (--hud=true)
+(ctrl-c) to exit
+```
+
+We expect that most users will want to hit (space) to open the browser,
+where they can browse logs, check deploy status, and trigger rebuilds.
+
+If you liked the terminal GUI and want to keep using it, you can hit the (h) key
+or start Tilt with `tilt up --hud=true`.
+
+If you just want to see logs, or don't have an interactive terminal, you can hit
+(s) or start Tilt with `tilt up --hud=false`.
+
+## What's Next?
+
+We currently have no plans to get rid of the terminal GUI.
+
+We may keep all 3 modes! Some people prefer a terminal, others prefer
+a browser UI, and a few even run everything in an Emacs buffer 👴.
```

Comment:
How about the less-explicitly-gendered :nerd_face: here?

Reply:
🤓